dice_ml/explainer_interfaces/explainer_base.py

```diff
@@ -191,43 +198,30 @@ def do_posthoc_sparsity_enhancement(self, final_cfs_sparse, query_instance, post
                         final_cfs_sparse = self.do_binary_search(diff, decimal_prec, query_instance, cf_ix, feature, final_cfs_sparse, current_pred)
 
             temp_preds = self.predict_fn_for_sparsity(final_cfs_sparse.iloc[[cf_ix]][self.data_interface.feature_names])
-            temp_preds = np.round(temp_preds.flatten().tolist(), 3)[0]
+            #temp_preds = np.round(temp_preds.flatten().tolist(), 3)[0]
             cfs_preds_sparse.append(temp_preds)
 
-        final_cfs_sparse[self.data_interface.outcome_name] = np.array(cfs_preds_sparse)
-
+        final_cfs_sparse[self.data_interface.outcome_name] = self.get_model_output_from_scores(cfs_preds_sparse)
+        #final_cfs_sparse[self.data_interface.outcome_name] = np.round(final_cfs_sparse[self.data_interface.outcome_name], 3)
```

Comment:
Shouldn't we use ```precisions = self.data_interface.get_decimal_precisions(output_type="dict")``` instead of rounding to 3?

Reply:
Good point, yes. I think it was 3 from before. Perhaps you can add the decimal precisions once this is merged?

Reply:
Sure